PyInstaller/utils/win32/icon.py

```diff
@@ -171,6 +172,8 @@ def CopyIcons(dstpath, srcpath):
     if isinstance(srcpath, str):
         # Just a single string, make it a one-element list.
         srcpath = [srcpath]
+    # Convert possible PathLike elements to strings to allow the splitter function to work.
+    srcpath = [str(path) if isinstance(path, os.PathLike) else path for path in srcpath]
```

Comment:
Why not just `srcpath = [str(path) for path in srcpath]`? `str(a)` with `a` already being a string should be a no-op...

Reply:
You are right, i updated the code (and fixed a bug i missed)